R/CompDb-methods.R

```diff
@@ -193,4 +193,18 @@ setMethod("deleteSpectra", signature(object = "CompDb"),
                   stop("'object' does not contain msms spectra")
               }
               object
-          })
\ No newline at end of file
+          })
+
+#' @inherit MetaboCoreUtils::mass2mz
+#' 
+#' @importFrom MetaboCoreUtils mass2mz
+#' 
+#' @export
+#' 
+#' @rdname CompDb
+setMethod("mass2mz", signature = c("CompDb"),
+          function(x, adduct){
+              exact_mass <- compounds(x)$exactmass
+              names(exact_mass) <- compounds(x)$formula
+              return(mass2mz(exact_mass, adduct))
```

Comment:
no need to use `return`, simpley have `mass2mz(exact_mass, adduct)` as the last line.